### DISCUSSION TITLE
[Web]The wrong position of the input box will block the edit area

### DISCUSSION THREAD

**Discussion by HUA616436641** [singerdmx/flutter-quill#661] [Web]
My issue is about [Web]

Reproduce step:
upgrade to latest stable version (2.8.1)
flutter pub get
flutter run

![QQ20220208-194028-HD](https://user-images.githubusercontent.com/17864476/152980396-747d04c4-33bd-4f63-a54b-5fa0dcc8bb39.gif)
![image](https://user-images.githubusercontent.com/17864476/152981118-9e15ec74-273c-4339-b61d-062027f1784d.png)

I found that the font style and height of the textarea are wrong.
I try to fix it by call `_textInputConnection!.setStyle `  in raw_editor_state_text_input_client_mixin.dart.The position is slightly corrected, but not completely.

**Comment by chifandeyu** [comment]
The same problem is now with using the pinyin input IME on the windows platform.

**Reply by chifandeyu** [reply]
```
[✓] Flutter (Channel stable, 3.3.8, on Microsoft Windows [版本 10.0.19044.2006], locale zh-CN)
[!] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
    ✗ Android SDK file not found: C:\Users\yuchang.xu\AppData\Local\Android\sdk\build-tools\32.0.0\aapt.
[✓] Visual Studio - develop for Windows (Visual Studio Community 2019 16.11.7)
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.74.2)
[✓] Connected device (1 available)
[✓] HTTP Host Availability
```
The position of the candidate window is wrong when input chinese use pinyin.

![xxzz9](https://user-images.githubusercontent.com/15946864/209131144-5790837f-53eb-458e-9da1-189bd6a7b5f2.jpg)
